### DISCUSSION TITLE
Når skal en discussion bli et issue?

### DISCUSSION THREAD

**Discussion by onordbo** [fremtind/jokul#2498] [Spørsmål]
## 🤔 Problemstilling
Nå som vi har gått over til vår nye oppgavemodell, og bruker discussions aktivt for å prioritere oppgaver og lage issues—har en ny problemstilling vist sitt ansikt. **Når kvalifiserer en discussion til å bli et issue?**

## 💡 Løsningsforslag/perspektiv
Mitt forslag/hypotese er at en discussion er detaljert nok til å bli en Issue når man kan være for eksempel:
- At man enten kan assigne ansvar til en utvikler og en designer
- Man kan enkelt estimere hvor lang tid det vil ta å lansere issuet (og man føler seg modig nok til å self-assigne oppgaven)
- Eller at det foreligger en Figma-skisse eller en pull-request med påbegynt arbeid som gjør det naturlig å aktivt spore arbeidet i en issue, i stedet for en discussion.

## 📖 Eksempler
Her er to forslag til eksempler på innslagspunkt/tidspunkt for når jeg synes det er naturlig en at en issue burde ha blitt opprettet på [tabeller](https://github.com/fremtind/jokul/discussions/2333#discussioncomment-1464449) og [tags](https://github.com/fremtind/jokul/discussions/2341#discussioncomment-1537561). _(Lenkene peker til kommentarer, fyi)_

**Comment by wkillerud** [comment]
Et annet eksempel jeg tenker er fint er fra sist forum: https://github.com/fremtind/jokul/issues/2493 ble opprettet fra https://github.com/fremtind/jokul/discussions/2454#discussioncomment-1612713

**Reply by onordbo** [reply]
@piofinn har timet et godt første steg i overgangen fra diskusjon til issue på kort. (#2345 👉 #2501)
